bare-metal/system_timer.c

```diff
@@ -35,19 +40,59 @@ typedef struct sys_timer_struct_s {
     ns_list_link_t link;
 } sys_timer_struct_s;
 
+#define TIMER_SLOTS_PER_MS			20
 #define TIMER_SYS_TICK_PERIOD       10 // milliseconds
-#define TIMER_SYS_TICK_SLOTS        (TIMER_SYS_TICK_PERIOD * 20) // 50us slots
+#define TIMER_SYS_TICK_SLOTS        (TIMER_SYS_TICK_PERIOD * TIMER_SLOTS_PER_SECOND) // 50us slots
```

Comment:
Yes should be SLOTS_PER_MS, though I guess this is not used now since the systimer period is given as milliseconds when starting it and that is converted to slots using TIMER_SLOTS_PER_MS.